lips/lip-8.md

```diff
@@ -125,12 +139,73 @@ The structure in this object can be a full pre-approval or just the fields of an
 }
 ```
 
+### ScopeObject
+
+In this object the initiator VASP declares its intent for the pre-approval, this can be one of two options:
+  1. Save the consumer sub-account for future transactions (save_sub_account)- this will enable the initiator VASP (merchant) to charge the sub-account in the future but will require the owner to approve the transaction. When using this option, amount limits are not required
+  2. Save the consumer sub-account and get a consent for future payments (consent) - this enables the initiator VASP (merchant) to charge the sub-account without any interaction with the owner. 
+Also, the scope limits the FundPullPreApprovalObject to certain parameters of time and amount. This object can be later mutated by the initiator VASP if needed, but any change requires the target VASP to approve the scope change.
+
+
+| Field         | Type  | Required?     | Description   |
+|-------        |------ |-----------    |-------------  |
+| type | str enum | Y | This can be either save_sub_account or consent |
+| expiration_timestamp | uint | Y | Unix timestamp indicating the time at which this pre-approval will expire - after which no funds pull can occur.  To expire an existing pre-approval early, this field can be updated with the current Unix timestamp. |
+| max_cumulative_amount | [ScopedCumulativeAmountObject](#scopedcumulativeamountobject) | N | Max cumulative amount that is approved for funds pre-approval.  This is the sum across all transactions that occur while utilizing this pre-approval. |
+| max_transaction_amount | [CurrencyObject](#currencyobject) | N | Max transaction amount that is approved for funds pre-approval.  This is the maximum amount that may occur in a single transaction while utilizing this funds pre-approval. |
+
+```
+{
+    "scope": {
+        "type": "consent",
+        "expiration_timestamp": 72322,
+        "max_cumulative_amount": {
+            "unit": "year",
+            "value": 1,
+            "max_amount": {
+                "amount": 100000,
+                "currency": "LBR",
+            }
+        },        
+        "max_transaction_amount": {
+            "amount": 100,
+            "currency": "LBR",
+        }
+    }
+}
+```
+
+### ScopedCumulativeAmountObject
+
+This object describes the scope of an amount
+
+| Field         | Type       | Required?    | Description   |
+|-------        |--------    |-----------   |-------------  |
+| unit          | str enum   | N            | One of: "day", "week", "month", "year" |
+| value         | int        | N            | Number of "unit"(s)   |
```

Comment:
what is difference between above 2 fields with "expiration_timestamp"? they are also confusing because different application may have different interpretation about what means of start of the day/week/month/year, hence not accurate.

Reply:
These fields are for limiting (scoping) the fund pulls from the consumer VASP. It can't be expressed using an expiration time, as it might be "not more than $100 a week". 
Regarding the interpretation of week start, duration, etc. - we think of it as a sliding time window, but one may think it's a calendar timeframe.
I will add a more detailed description of it.

Reply:
unit - reset_period?
value - amount_per_period

also I would probably move max_amount and currency to the same level since currency refers to both...